tests/Unit/Controllers/SupportControllerTest.php

```
@@ -0,0 +1,121 @@
+        if (session_status() === PHP_SESSION_NONE) {
+            @session_start();
+        }
+        $_SESSION = ['_csrf_token' => 'csrf-test'];
+        $this->actingAs($this->user);
+    }
+
+    protected function tearDown(): void
+    {
+        $_SESSION = [];
+        parent::tearDown();
+    }
```

PHPMD (2.15.0)

[error] tearDown accesses the super-global variable $_SESSION. (undefined)

(Superglobals)